projects/ad-acevsecrdset-sl/src/self_test/self_test.c

```diff
@@ -441,6 +470,26 @@ static int self_test_pilot(struct stout *stout)
 	return 0;
 }
 
+#if defined(REV_D)
+/**
+ * @brief Startup test for PE upstream presence
+ * @param stout - state  machine descriptor
+ * @return 0 in case of success, error code otherwise
+ */
+int self_test_PE_upstream(struct stout *stout)
```

Comment:
not sure about the previous implementation of the project, but snake case convention is applied.